### ISSUE TITLE
hello MatteoAntoniazzi im new to python and need your help

### ISSUE THREAD

**Issue by RangXuan** [MatteoAntoniazzi/MAPF#2]
I don't know how to implement your code when I launch launcher.py   
I don't know how to see the simulator result

**Comment by RangXuan** [comment]
There are only parameter settings in the launcher.py file, no running, and visualization

**Comment by MatteoAntoniazzi** [comment]
Sorry if I answer now. In order to run my code you can just download it, create the virtual environment through the 'requirements.txt' file and launch the 'launcher.py' file.  In the launcher a 'StartMenu' object is created which takes care of creating the graphic menu for the simulation. Let me know if you need anything else.

**Comment by RangXuan** [comment]
> Sorry if I answer now. In order to run my code, you can just download it, create the virtual environment through the 'requirements.txt' file and launch the 'launcher.py' file. In the launcher, a 'StartMenu' object is created which takes care of creating the graphic menu for the simulation. Let me know if you need anything else.

when the  GUI's size is [1300 700] [ 600 700] 800, on my PC I can not see the button of 'PREPARE'
and then I resize the GUI's size to 1300 800 600 700 800, I can see it
thanks for your reply  and your contribution to MAPF

**Comment by MatteoAntoniazzi** [comment]
Yes, that is the correct way. It can be caused by a different screen resolution. Thanks for your feedbacks.
